codegen/templates/schema.tmpl

```diff
@@ -4,10 +4,13 @@
 
 package {{.Package}}
 {{$sfl := len .SelectableFields}}
+{{$needsFmt := false}}{{range .SelectableFields}}{{if ne .Type "string"}}{{$needsFmt = true}}{{end}}{{end}}
 import (
     {{if gt $sfl 0}}
-    "fmt"
     "errors"
+    {{if $needsFmt}}
```

Comment:
Can we change this to 
```
"errors"{{if $needsFmt}}
"fmt"{{end}}
```
So that we don't end up with extra line breaks that will upset `gofmt`?